src/rocm_docs/core.py

```diff
@@ -9,8 +9,11 @@
 from abc import ABC, abstractmethod
 from pathlib import Path
 from typing import Any, Callable, Dict, Generic, List, Type, TypeVar
-from bs4 import BeautifulSoup
+import importlib_resources
```

Comment:
This package is only a dependency with python < 3.9, see the imports in doxygen.py on how to handle theis based on version.

Reply:
Fixed based on doxygen.py in [a84f63f](https://github.com/RadeonOpenCompute/rocm-docs-core/pull/139/commits/a84f63fcfe0e77d60591a27d2e882c3bd812e212)